core/retries/src/main/java/software/amazon/awssdk/retries/DefaultRetryStrategy.java

```diff
@@ -59,7 +79,7 @@ public static StandardRetryStrategy.Builder standardStrategyBuilder() {
      *                    .build();
      * </pre>
      */
-    public static AdaptiveRetryStrategy.Builder adaptiveRetryStrategyBuilder() {
+    public static AdaptiveRetryStrategy.Builder adaptiveStrategyBuilder() {
```

Comment:
Why drop `Retry`?

Reply:
Oh, that was added by accident, all the rest does not have the retry work in there.